Program.cs

```diff
@@ -76,5 +108,121 @@ public static async Task<ICollection<Entry>> GetFolderChildren(IRepositoryApiCli
             }
             return children.Value;
         }
+
+        public static async Task<Entry> CreateFolder(IRepositoryApiClient client, string repoId)
+        {
+            const string newEntryName = ".Net sample project folder";
+            PostEntryChildrenRequest request = new PostEntryChildrenRequest();
+            request.EntryType = PostEntryChildrenEntryType.Folder;
+            request.Name = newEntryName;
+            Console.WriteLine("\nCreating sample project folder...");
+            Entry result = await client.EntriesClient.CreateOrCopyEntryAsync(repoId, rootFolderEntryId, request, true);
+            tempSampleProjectFolderId = result.Id;
+            return result;
+        }
+
+        public static async Task ImportDocument(IRepositoryApiClient client, string repoId, int folderEntryId, string sampleProjectFileName)
+        {
+            int parentEntryId = folderEntryId;
+            string fileName = sampleProjectFileName;
+            var electronicDocument = GetFileParameter();
+            var request = new PostEntryWithEdocMetadataRequest();
+            Console.WriteLine("\nImporting a document into the sample project folder...");
+            var result = await client.EntriesClient.ImportDocumentAsync(repoId, parentEntryId, fileName, autoRename: true, electronicDocument: electronicDocument, request: request).ConfigureAwait(false);
+            tempEdocEntryId = result.Operations.EntryCreate.EntryId;
```

Comment:
could return newly imported document entryId? So in Main(), it could just use returned entryId to get GetEntryContentType

Reply:
ok done for all 3 sample project